15-lhcb-dev.md

```diff
@@ -27,20 +27,20 @@ Successfully created the local project DaVinciDev in .
 To start working:
 
   > cd ./DaVinciDev
-  > getpack MyPackage vXrY
+  > git lb-use DaVinci
+  > git lb-checkout DaVinci/vXrY MyPackage
 
 then
 
   > make
   > make test
-  > make QMTestSummary
```

Comment:
This is just the output obtained by running `lb-dev --name DaVinciDev DaVinci v41r2` (it seems to be the same in v40r2 as well).